### PR TITLE
Fixed assign_current method

### LINKED ISSUE #619: Enable 'Maxwell3d.assign_current' Method to work with Magnetostatic Solver
Unless I'm mistaken the titled method does not work with the magnetostatic solver
image

### PR DESCRIPTION
**Pull request by maxcapodi78**
Fixed #619

### CODE REVIEW

**Comment by maxcapodi78** [comment]
@beliaev-maksim This method was not working in Magnetostatic analysis because AEDT API doesn't expect Phase argument for that solution type. I've also fixed it for other solutions like ElectricTransient and DCConduction

File: pyaedt/maxwell.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def assign_current(self, object_list, amplitude=1, phase="0deg", solid=True, swa
         object_list : list
             List of objects to assign the current source to.
         amplitude : float, optional
-            Voltage amplitude in mV. The default is ``1``.
+            Voltage amplitude in mA. The default is ``1``.
```
Comment (beliaev-maksim) [inline pyaedt/maxwell.py:217]: voltage in amperes?

File: pyaedt/maxwell.py [hunk 2]
```diff
@@ -247,20 +248,20 @@ def assign_current(self, object_list, amplitude=1, phase="0deg", solid=True, swa
...
-                        "Phase": phase,
-                        "IsSolid": solid,
-                        "Point out of terminal": swap_direction,
                     }
-                )
+                    )
+            if self.solution_type not in ["Magnetostatic", 'DCConduction', 'ElectricTransient']:
+                props["Phase"] = phase
+            if self.solution_type not in ['DCConduction', 'ElectricTransient']:
+                props["IsSolid"] = solid
```
Comment (beliaev-maksim) [inline pyaedt/maxwell.py:263]:
```suggestion
            if self.solution_type not in ["Magnetostatic", 'DCConduction', 'ElectricTransient']:
                props["Phase"] = phase
                if self.solution_type not in ['DCConduction', 'ElectricTransient']:
                    props["IsSolid"] = solid
```

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]

File: pyaedt/maxwell.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/maxwell.py:217]: @beliaev-maksim Correct. it comes from assign_voltage method. I'll fix it

**Pull request merged**